tests/fixtures/saas/salesforce_fixtures.py

```diff
@@ -219,4 +220,17 @@ def salesforce_create_erasure_data(
     assert campaign_members_response.ok
     campaign_member_id = campaign_members_response.json()["id"]
 
-    yield contact_id, lead_id, case_id, account_id, campaign_member_id
+    yield account_id, contact_id, case_id, lead_id, campaign_member_id
+
+    # cleanup data by doing a full deletion instead of just masking
+    case_response = requests.delete(
+        url=f"{base_url}/services/data/v54.0/sobjects/Case/{case_id}", headers=headers
+    )
+    assert case_response.ok
+
+    account_response = requests.delete(
+        url=f"{base_url}/services/data/v54.0/sobjects/Account/{account_id}",
+        headers=headers,
+    )
+
+    assert account_response.ok
```

Comment:
i assume it's not feasible to actually confirm with salesforce that the data was deleted? (i.e. it takes too much time to process the delete)